packages/components/src/components/hds/advanced-table/index.ts

```diff
@@ -311,6 +312,16 @@ export default class HdsAdvancedTable extends Component<HdsAdvancedTableSignatur
     return density;
   }
 
+  get hasStickyHeader(): boolean {
```

Comment:
[question] this logic allows the condition where the table has a max height but the header is not sticky; is it intentional (I imagine so, given the integration tests below)? and is it accessible (I also imagine so, but I wanted to double check)

Reply:
Yup! We just wanted to give people an escape hatch to not have the sticky header + set a max height.